api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -89,7 +95,11 @@ public static Tagger getTagger() {
    * @throws IllegalStateException if a specified provider is not found
    */
   @Nullable
-  private static <T> T loadSpi(Class<T> providerClass) {
+  private static <T> T loadSpi(Class<T> providerClass, ClassLoader classLoader) {
+    if (classLoader.getParent() != null) {
+      return loadSpi(providerClass, classLoader.getParent());
+    }
+
     String specifiedProvider = System.getProperty(providerClass.getName());
     ServiceLoader<T> providers = ServiceLoader.load(providerClass);
```

Comment:
`ServiceLoader.load(providerClass)` doesn't use `classloader`.  Should this be `ServiceLoader.load(providerClass, classloader)`?

Reply:
yes it should